tests/test_metadata.py

```diff
@@ -393,6 +406,18 @@ def test_general_labels(
             assert "BCI" in labels[f"{prefix}.title"]
 
 
+@pytest.mark.parametrize("container", ALL_CONTAINERS, indirect=True)
+def test_support_end_in_future(container: ContainerData):
+    labels = container.inspect.config.labels
+    if "com.suse.supportlevel.until" in labels:
+        support_end = datetime.datetime.fromisoformat(
+            labels["com.suse.supportlevel.until"]
+        )
```

Comment:
This breaks for the SUSE AI containers, as they have no valid eol date

Reply:
moved the check to a separate PR